src/js/TuningSliders.js

```diff
@@ -156,6 +156,9 @@ TuningSliders.updateExpertModeSlidersDisplay = function() {
     $('.advancedSliderPitchPIGain').toggle(rpIGain || this.expertMode);
     $('.advancedSliderMaster').toggle(master || this.expertMode);
 
+    $('.sliderGyroFilter').toggleClass('disabledSliders', !this.sliderGyroFilter || (gyro && !this.expertMode));
+    $('.sliderDTermFilter').toggleClass('disabledSliders', !this.sliderDTermFilter || (dterm && !this.expertMode));
```

Comment:
Disabling expert mode Dterm filters slider is disabling completly now.

Reply:
Current range is outside allowed non-expert mode sliders. For that reason they are being disabled.

Reply:
The advice message is shown here. How did you reproduce?

Reply:
Im refering to PiD sliders here.

Reply:
For filters side, dterm sliders disabled with non expert mode, but guro sliders remains enabled.

Reply:
About the PID sliders advice how to reproduce as it's toggles ok using the expert mode?

EDIT: Pushed a small addition to enable the expertSettingsDetectedNote to make it clear for anyone what's going on the filters tab.

Reply:
@Asizon Perhaps it's due to having two classes with same name. Pushed again using sub class. Please check again.